examples/todo/todo.py

```diff
@@ -4,15 +4,40 @@
 from flask_rebar import Rebar, errors, HeaderApiKeyAuthenticator
 from flask_rebar.validation import RequestSchema, ResponseSchema
 
+from flask_rebar import Tag,SwaggerV2Generator
 
 rebar = Rebar()
-registry = rebar.create_handler_registry()
+
+# add a description of the tags we're using further down (will be output to the generated swagger)
+# a bit reduntant, given the simplicity of the code, but you get the idea :) 
+generator = SwaggerV2Generator(
+    tags=[
+        Tag(
```

Comment:
No need for self hating;) I was struggling with that a bit myself, I thought these would make more sense than just a dummy tag. But since I only wanted it to demonstrate how to use the tagging feature, I don't feel too strongly about that.